### PR TITLE
Updated balsamiq-mockups

### PR DESCRIPTION
**Pull request by tsparber**

### CODE REVIEW

File: Casks/balsamiq-mockups.rb [hunk 1]
```diff
@@ -9,5 +9,10 @@
 
   app "Balsamiq Mockups #{version.to_i}.app"
```
Comment (vitorgalvao) [inline Casks/balsamiq-mockups.rb]: Could be `#{version.major}`.

File: Casks/balsamiq-mockups.rb [hunk 2]
```diff
@@ -9,5 +9,10 @@
...
 
-  zap :delete => '~/Library/Preferences/BalsamiqMockups3'
+  zap :delete => [
+                   '~/Library/Preferences/BalsamiqMockups3',
+                   # TODO: expand/glob for '~/Library/Preferences/BalsamiqMockups3.*',
```
Comment (vitorgalvao) [inline Casks/balsamiq-mockups.rb]: Why not go all out and do `BalsamiqMockups*` instead, also taking care of future versions?

**Comment by adityadalal924** [comment]
Thank you for the contribution. It was merged directly as 8a784190306c031807223fdc21cf81c4fb295aa0 to keep commit history cleaner. Your contribution is still credited to you.

File: Casks/balsamiq-mockups.rb [hunk 3]
```diff
@@ -7,7 +7,14 @@
...
-  app "Balsamiq Mockups #{version.to_i}.app"
+  app "Balsamiq Mockups #{version.major}.app"
 
-  zap :delete => '~/Library/Preferences/BalsamiqMockups3'
+  zap :delete => [
+                   # TODO: expand/glob for '~/Library/Caches/BalsamiqMockups#{version.major}.*',
+                   # TODO: expand/glob for '~/Library/Saved Application State/BalsamiqMockups#{version.major}.*',
+                 ],
+      :trash  => [
+                   '~/Library/Preferences/BalsamiqMockups#{version.major}',
```
Comment (tsparber) [inline Casks/balsamiq-mockups.rb]: Reverting in #17847. Using `#{version.major}` seems not working in `:trash`

```
testMac:~ tommy$ brew cask zap balsamiq-mockups
==> Implied "brew cask uninstall balsamiq-mockups"
==> Removing App symlink: '/Users/tommy/Applications/Balsamiq Mockups 3.app'
==> Dispatching zap stanza
==> Running zap process for balsamiq-mockups; your password may be necessary
==> Removing files: ["~/Library/Preferences/BalsamiqMockups\\\#{version.major}"]
Password:
==> Removing files: ["~/Library/Preferences/BalsamiqMockups\\\#{version.major}", "~/Library/Prefe
==> Removing all staged versions of Cask 'balsamiq-mockups'
```

Comment (vitorgalvao) [inline Casks/balsamiq-mockups.rb]: It isn’t working because single quotes are used, instead of double quotes.

Comment (tsparber) [inline Casks/balsamiq-mockups.rb]: Whoops, thank you for fixing it.

**Pull request closed without merging**